docs/cli-existing-project.md

```diff
@@ -145,60 +153,41 @@ We'll explain the Sphinx modifier in a later guide.
 
 Remove any `vm.startBroadcast` and `vm.broadcast` calls from your deployment script. Broadcasting is no longer required because you won't be executing your deployment from the CLI.
 
-### e. Handle new sender address
-
-When using Sphinx, your deployment will be executed from your Gnosis Safe. In other words, the `msg.sender` of your transactions will be your Gnosis Safe. You may need to update your script if it relies on a particular sender address. If you need to access your Gnosis Safe address, you can fetch it in your script using `safeAddress()`.
-
-For example, you may need to:
-- Update hardcoded contract addresses
-- Assign permissions using your Gnosis Safe address
-
-### f. Add configuration options
+### e. Configure project name
 
-There are a few configuration options that you must specify inside a `configureSphinx()` function in your deployment script. The options all exist on the `sphinxConfig` struct, which is inherited from `Sphinx.sol`.
+In your script, you must specify the name of the project you would like to deploy with. This is configured in a `configureSphinx()` function in your deployment script. The `projectName` field exists on the `sphinxConfig` struct which is inherited from `Sphinx.sol`.
 
 Copy and paste the following `configureSphinx()` function template into your script:
 
 ```sol
 function configureSphinx() public override {
-  sphinxConfig.owners = [<your address>];
-  sphinxConfig.orgId = <Sphinx org ID>;
-  sphinxConfig.projectName = "My_First_Project";
-  sphinxConfig.threshold = 1;
+  sphinxConfig.projectName = "<your_project_name>";
 }
```

Comment:
We could simplify this even further by having the user define the project name as a top-level constant. However, that is somewhat non-trivial to handle, so I left it out of this PR. I've [created a ticket for this](https://linear.app/chugsplash/issue/CHU-1017/switch-to-configuring-script-project-name-as-a-top-level-constant).